src/Collection/Collection.model.ts

```diff
@@ -91,16 +110,28 @@ export class Collection extends Model<CollectionAttributes> {
         ? [
             CurationStatusFilter.PENDING,
             CurationStatusFilter.APPROVED,
+            CurationStatusFilter.REJECTED,
           ].includes(status)
-          ? SQL`collection_curations.status = ${status} AND collections.contract_address = ANY(${remoteIds})`
-          : status === CurationStatusFilter.REJECTED // Rejected not a single curation in approved
-          ? SQL`collection_curations.status = ${status} AND collections.contract_address = ANY(${remoteIds})`
+          ? SQL`(collection_curations.status = ${status} AND (collections.contract_address = ANY(${remoteIds}) OR ${raw(
+              isThirdParty
+            )})) OR (collection_curations.id is NULL AND collections.contract_address = ANY(${remoteIds}))`
           : status === CurationStatusFilter.TO_REVIEW // To review: Not assigned && isApproved false from the contract
-          ? SQL`collection_curations.assignee is NULL AND collections.contract_address = ANY(${remoteIds})`
+          ? SQL`collection_curations.assignee is NULL AND ((${raw(
+              isThirdParty
+            )}) OR (collections.contract_address = ANY(${remoteIds}) AND ${raw(
+              isStandard
+            )}))`
           : status === CurationStatusFilter.UNDER_REVIEW // Under review: isApproved false from the contract OR it's assigned & has pending curation
           ? SQL`collection_curations.assignee is NOT NULL AND collection_curations.status = ${CurationStatusFilter.PENDING}`
           : SQL``
         : undefined,
+      type
+        ? type === CollectionTypeFilter.STANDARD
+          ? SQL`${raw(isStandard)}`
```

Comment:
```suggestion
          ? isStandard
```
these won't be needed anymore if we wrap the variables with the `SQL`